.github/workflows/ci-elasticsearch.yml

```diff
@@ -50,10 +44,24 @@ jobs:
       run: make install-ci
 
     - uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
+    
+    - name: Set up Elasticsearch
+      run: |
+           docker-compose -f  ./.github/workflows/docker-compose-elasticsearch-v${{ matrix.major-version }}.yml up -d
```

Comment:
we don't run these from the workflow, they need to be run from the script, so that the integration tests are repeatable locally